### PR TITLE
AddisonAdcock-the-process-meme

### PR DESCRIPTION
**Pull request by Soyokaze-42**
AddisonAdcock's meme for the-process

### CODE REVIEW

**Comment by Soyokaze-42** [comment]
Good job!

File: _posts/2023-01-30-AddisonAdcock-the-process-meme.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Simple vs. Modern Operating Systems
+author:
```
Comment (Soyokaze-42) [inline _posts/2023-01-30-AddisonAdcock-the-process-meme.md:3]: I think I lost a comment somewhere...

The author should be Addison's short_name. Fix that and I can merge.

Comment (AddisonAdcock) [reply _posts/2023-01-30-AddisonAdcock-the-process-meme.md]: I just committed and added my name. I think that was what was missing?

Comment (Soyokaze-42) [reply _posts/2023-01-30-AddisonAdcock-the-process-meme.md]: yep! Thanks.

**Pull request merged**